src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -260,6 +315,41 @@ export default class AmalgamatingBusinesses extends Mixins(AmalgamationMixin, Co
     ]
   }
 
+  /** Whether the business table is valid. */
+  get businessTableValid (): boolean {
+    if (this.isAmalgamationFilingRegular) {
+      return (this.haveAmalgamatingBusinesses && this.allOk)
+    } else {
+      return (this.haveHoldingBusiness && this.haveAmalgamatingBusinesses && this.allOk)
+    }
+  }
+
+  /**
+   * Whether we have a holding/primary business in the table.
+   * (Only used for short-form amalgamations.)
+   */
+  get haveHoldingBusiness (): boolean {
```

Comment:
Would it make sense to have this and the function below in the mixin Sev?

Reply:
Good point. Yes, these are "general values" and it makes sense for them to be in the amalgamation mixin. On the other hand, these computed properties are not used elsewhere and it makes sense to limit their scope. I could go either way.

Karim, others, let me know your thoughts.

Reply:
True. Both make sense. My thought was also in case we need to re-use them later for whatever reason in other files, we can just grab them then (don't think that'll happen).

Sev, I have no strong opinion on this. I'm good to keep as is. If the need rises to re-use them, we can just move them to the mixin.